File: 240205/피보나치 수/fibonacci-number.cpp
#include <iostream>

using namespace std;

int arr[50];

int main() {
    int n;
    cin>>n;
    arr[0] = 0;
    arr[1] = 1;
    arr[2] = 1;

    for(int i = 3;i<=n;i++){
        arr[i] = arr[i - 1] + arr[i - 2];
    }

    cout<<arr[n]<<endl;

    return 0;
}

File: 240205/피보나치 수/fibonacci-number.swift
var n = Int(readLine()!)!

var arr = Array(repeating: 0, count: 50)
arr[0] = 0
arr[1] = 1
arr[2] = 1


for i in stride(from: 2, to: n, by: 1) {
    arr[i] = arr[i - 1] + arr[i - 2]
}

print(arr[n])